FRW/UI/Extension/DispatchQueue.swift

```diff
@@ -9,6 +9,7 @@ import SwiftUI
 import UIKit
 
 extension DispatchQueue {
+    //TODO: this is not a sync method
```

Comment:
@lmcmz fyi there's a few places calling this method, not sure if this is causing issues elsewhere but it was causing a UI hiccup during this refactor.